frontend/sw360-portlet/src/main/webapp/html/preferences/view.jsp

```diff
@@ -0,0 +1,104 @@
+<%--
+  ~ Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+  ~
+  ~ SPDX-License-Identifier: EPL-1.0
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+--%>
+<%@ page import="org.eclipse.sw360.datahandler.thrift.users.User" %>
+
+<%@include file="/html/init.jsp" %>
+<%-- the following is needed by liferay to display error messages--%>
+<%@include file="/html/utils/includes/errorKeyToMessage.jspf"%>
+<portlet:defineObjects/>
+<liferay-theme:defineObjects/>
+
+<jsp:useBean id="sw360User" type="org.eclipse.sw360.datahandler.thrift.users.User" scope="request"/>
+<jsp:useBean id="eventsConfig" type="java.util.Map<java.lang.String, java.util.List<java.util.Map.Entry<java.lang.String, java.lang.String>>>" scope="request"/>
+
+<portlet:actionURL var="saveUserPreferencesURL" name="savePreferences"/>
+
+<link rel="stylesheet" href="<%=request.getContextPath()%>/css/sw360.css">
+<script src="<%=request.getContextPath()%>/webjars/jquery/1.12.4/jquery.min.js" type="text/javascript"></script>
+
+<div id="header"></div>
+<p class="pageHeader"><span class="pageHeaderBigSpan">User Preferences</span></p>
+<div id="content">
+    <table class="table info_table" id="readOnlyUserData" title="SW360 User Data">
+        <thead>
+            <tr>
+                <th colspan="2">SW360 User</th>
+            </tr>
+        </thead>
+        <tr>
+            <td>Name:</td>
+            <td><sw360:out value="${sw360User.fullname}"/></td>
+        </tr>
+        <tr>
+            <td>Id (E-Mail):</td>
+            <td><sw360:DisplayUserEmail email="${sw360User.email}" bare="true"/></td>
+        </tr>
+        <tr>
+            <td>Group:</td>
+            <td><sw360:out value="${sw360User.department}"/></td>
+        </tr>
+        <tr>
+            <td>External Id:</td>
+            <td><sw360:out value="${sw360User.externalid}"/></td>
+        </tr>
+        <tr>
+            <td>Role:</td>
+            <td><sw360:DisplayEnum value="${sw360User.userGroup}"/></td>
+        </tr>
+    </table>
+    <form action="<%=saveUserPreferencesURL%>" id="preferences_form" method="post">
```

Comment:
What do you think about changing the order of labels and inputs for all following checkboxes? This might make the page a lot more steady.

Reply:
good point